ZeldaFullEditor/Data/TileUndo.cs

```diff
@@ -37,7 +37,7 @@ public void Restore(SceneOW scene)
                     int superY = (mouseYDown + y) / 32;
                     int mapId = (superY * 8) + superX + scene.ow.WorldOffset;
                     usedTiles[x + mouseXDown, y + mouseYDown] = savedTiles[i];
-                    scene.ow.AllMaps[mapId].BuildMap();
+                    //scene.ow.AllMaps[mapId].BuildMap();
```

Comment:
if we don't plan to do this anymore, we should delete the code or explain why it is commented out now